src/js/actions/documents.js

```diff
@@ -763,8 +768,7 @@ define(function (require, exports) {
         reads: [locks.JS_TOOL],
         writes: [locks.JS_APP, locks.PS_APP],
         transfers: ["layers.resetLinkedLayers", historyActions.queryCurrentHistory,
-            ui.updateTransform, toolActions.select, "panel.cloak", guideActions.queryCurrentGuides,
-            toolActions.changeVectorMaskMode, updateDocument],
+            ui.updateTransform, toolActions.select, toolActions.changeVectorMaskMode, updateDocument],
```

Comment:
I just removed two dead transfers.  And luckily I didn't have to add `historyActions.queryCurrentHistory` because it was already sitting here rotting.  hooray.